model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -329,8 +333,33 @@ def _get_next_perf_analyzer_config(self, model_variant_name: str,
             self._models[model_num].perf_analyzer_flags())
         return perf_analyzer_config
 
-    def _print_debug_logs(self, measurements: List[Union[RunConfigMeasurement,
-                                                         None]]) -> None:
+    def _create_default_run_config(self) -> RunConfig:
+        default_run_config = RunConfig(self._triton_env)
+
+        for model in self._models:
+            default_model_config = BaseModelConfigGenerator.make_model_config(
+                param_combo={},
+                model=model,
+                model_variant_name_manager=self._model_variant_name_manager)
+
+            default_perf_analyzer_config = PerfAnalyzerConfig()
+            default_perf_analyzer_config.update_config_from_profile_config(
+                default_model_config._model_config.name, self._config)
```

Comment:
don't reach in to private variable.

default_model_config.get_field('name')

Reply:
Fixed